backend/api/contacts/src/test/java/co/airy/core/contacts/ImportContactsTest.java

```diff
@@ -72,77 +60,57 @@ void beforeEach() throws Exception {
         webTestHelper.waitUntilHealthy();
     }
 
-    @NoArgsConstructor
-    private class ContactsList implements RunnableTest {
-
-        @Getter
-        private String listContent;
-
-        public void test() throws Exception {
-            listContent = webTestHelper.post("/contacts.list")
-                .andExpect(status().isOk())
-                .andExpect(jsonPath("$.data", hasSize(4)))
-                .andReturn().getResponse().getContentAsString();
-        }
-
-    }
-
     @Test
     void canImportContacts() throws Exception {
         final List<CreateContactPayload> payload = mockContactsListPayload();
 
-        final String importContent = webTestHelper.post(
-                "/contacts.import",
-                objectMapper.writeValueAsString(payload))
-            .andExpect(status().isCreated()).andReturn().getResponse().getContentAsString();
+        final ResultActions importActions = webTestHelper.post(
+                        "/contacts.import",
+                        objectMapper.writeValueAsString(payload))
+                .andExpect(status().isCreated());
 
-        List<ContactResponsePayload> contactsResp = objectMapper.readValue(
-                importContent,
-                new TypeReference<List<ContactResponsePayload>>() {});
-
-        ContactsList cl = new ContactsList();
-        retryOnException(cl, "Not able to get contacts list");
-
-        final ListContactsResponsePayload contactsList = objectMapper.readValue(cl.getListContent(), ListContactsResponsePayload.class);
```

Comment:
Small refactor. Generally we try to avoid using our own payload POJOs to test APIs, because that way we don't check what clients actually receive.